dev-libs/mongo-c-driver/mongo-c-driver-1.6.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils
```

Comment:
Is eutils actually used? If not, you don't have to push a fix — just ACK and I'll fix it while merging.